### DISCUSSION TITLE
create my own command and release it on pip?

### DISCUSSION THREAD

**Discussion by spxneo** [fastapi/typer#874] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
is there a way to compile this to my own custom command instead of having my user run `typer main.py` ?
```

### Description

meaning i want my user to install via 

`pip install mytool`

and then immediately call

`mytool --custom options`

### Operating System

macOS

### Typer Version

1

### Python Version

3

**Comment by YuriiMotov** [comment, marked as answer]
Explained here: https://typer.tiangolo.com/tutorial/package/
